clustermesh/clustermesh.go

```diff
@@ -1425,48 +1426,75 @@ func log(format string, a ...interface{}) {
 	fmt.Fprintf(os.Stdout, format+"\n", a...)
 }
 
+func parseAnnotations(annotations []string) map[string]interface{} {
+	annotationsMap := make(map[string]interface{})
+	for _, annotation := range annotations {
+		splitAnnotation := strings.SplitN(annotation, "=", 2)
+		annotationsMap[splitAnnotation[0]] = splitAnnotation[1]
+	}
+	return annotationsMap
+}
+
+func mergeAnnotationsMap(sourceAnnotationsMap map[string]interface{}, targetAnnotationsMap map[string]interface{}) {
+	if targetAnnotationsMap != nil {
+		for k, v := range targetAnnotationsMap {
+			sourceAnnotationsMap[k] = v
+		}
+	}
+}
+
 func generateEnableHelmValues(params Parameters, flavor k8s.Flavor) (map[string]interface{}, error) {
 	helmVals := map[string]interface{}{
 		"clustermesh": map[string]interface{}{
+			"clusters":     []string{},
```

Comment:
Did you intend to include this line? There's a `clustermesh.config.clusters` value, but doesn't seem related to this change.

Reply:
Good catch! My bad, this wasn't meant to be here at all. 🙈